docs/content/concepts/automation/declarative-automation/customizing-automation-conditions.mdx

```diff
@@ -0,0 +1,287 @@
+- **Events** are transient and reflect something that may only be true for an instant. For example, the `AutomationCondition.newly_updated()` condition will be true only if an asset partition was materialized since the previous evaluation.
+
+Using the `<A>.since(<B>)` operator, you can create conditions that detect if one event has happened more recently than another. Think of this as converting two events to a status - in this case, `A has occurred more recently than B` - as this will stay true for some period of time. This operator becomes true whenever `<A>` is true, and will remain true until `<B>` is also true.
+
+Conversely, it can also be useful to convert statuses to events. For example, the default `eager()` condition ensures that Dagster only tries to materialize a missing asset partition once using the following sub-condition:
+
+```python
+from dagster import AutomationCondition
+
+AutomationCondition.missing().newly_true().since(
+    AutomationCondition.newly_requested() | AutomationCondition.newly_updated()
+)
+```
+
+By using the `<A>.newly_true()` operator, you can turn the status of _"being missing"_ into a single event, specifically the point in time where an asset partition entered the _missing_ state. From there, you can ensure that an asset is materialized only once in response to detecting a missing partition.
+
+---
+
+## Using conditions to chain runs
+
+Dagster can group the execution of multiple assets into a single, logical run. For example, imagine you have a series of dependent assets, each with an `AutomationCondition.eager()` condition. When you update the first asset in the chain, the desired behavior is typically to have all downstream assets grouped into a single run, rather than executing each asset in order in individual run.
+
+To create this scenario, you can use `AutomationCondition.will_be_requested()`. Because each <PyObject object="AutomationCondition" /> is evaluated in order, you can query if an upstream asset will be requested on the current tick. For example:
+
+```python
+from dagster import AutomationCondition
+
+any_parent_missing = AutomationCondition.any_deps_match(
+    AutomationCondition.missing() & ~AutomationCondition.will_be_requested()
+)
+```
+
+---
+
+## Examples
+
+### Ignoring specific parents
+
+By default, `AutomationCondition.eager()` will materialize in response to **any** parent update, and not materialize if **any** parent is missing.
+
+### Updating in response to code version changes
+
+When you update the an asset's code version, you'll typical want to materialize the asset to reflect the change. To do this, you can use `AutomationCondition.code_version_changed()`:
+
+```jsx
+automation_condition = AutomationCondition.code_version_changed();
+```
+
+This condition will immediately materialize any asset as soon as a code change is detected.
```

Comment:
The Notion draft made it seem like there was supposed to be more here, but we can do a fast-follow here if needed.